### DISCUSSION TITLE
versus Crashlytics -- can I remove Crashlytics and just use Sentry?

### DISCUSSION THREAD

**Discussion by timfong888** [getsentry/support#36] [Support] [answered]
I have read mixed things in terms of whether there are types of crashes only Crashlytics can catch.

But I keep getting errors when trying to install Crashlytics (the app crashes and I keep getting a `exception 'NSInvalidArgumentException', reason: '-[FIRExceptionModel setOnDemand:]:`

But going through SO and looking at the Crashlytics DSYM upload scripts....doesn't fix it, so I would like to just remove it for now since the crash reporting isn't critical now...

But I haven't had issues with Sentry yet...so hoping Sentry crash reporting can replace Crashlytics altogether.

**Comment by iwithrow** [comment]
Our goal is that you can replace Crashlytics with Sentry with confidence. Try it and let us know how it goes. 

cc @bruno-garcia

**Reply by bruno-garcia** [reply]
Sorry I can't keep up with GitHub notifications.

**Comment by bruno-garcia** [comment, marked as answer]
I'm unaware of any type of crash or error that Crashlytics can capture and Sentry doesn't. If anything I believe to be the other way around, unless Crashlytics captures [iOS App Hangs](https://docs.sentry.io/platforms/apple/configuration/app-hangs/), [iOS OutOfMemory](https://docs.sentry.io/platforms/apple/configuration/out-of-memory/)?
